### PR TITLE
Write cni config to /etc/cni/net.d

### LINKED ISSUE #43: Rename CNI's "mynet" to something more descriptive
kubeadm-nspawn hard codes "mynet" but that name does not give any hint that it is about kubeadm-nspawn. We should find a better name.

Capturing comment https://github.com/kinvolk/kubeadm-nspawn/issues/35#issuecomment-308473076:

> we could suggest a better name than "mynet" (hardcoded in pkg/cnispawn/netns.go)... it could be "nspawnnet".

### PR DESCRIPTION
**Pull request by robertgzr**
Also updates the code according to upstream changes

Closes #43

### CODE REVIEW

**Comment by robertgzr** [comment]
@alban Maybe updating glide should be in here along with the stuff from #41

File: pkg/bootstrap/cninet.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+const NspawnNetPath string = "/etc/cni/net.d/10-kubeadm-nspawn-net.conf"
+const NspawnNetConf string = `
+{
+    "cniVersion": "0.2.0",
+    "name": "mynet",
```
Comment (alban) [inline pkg/bootstrap/cninet.go]: stil called mynet?

Comment (robertgzr) [inline pkg/bootstrap/cninet.go]: Oops. Will be fixed

File: README.md [hunk 2]
```diff
@@ -74,51 +74,14 @@ cd cluster/images/hyperkube
...
-git clone git@github.com:containernetworking/cni.git
-```
-
-Then you can build CNI by doing:
+### Get CNI plugins
 
 ```
-cd cni
-./build
+get get -u github.com/containernetworking/plugins/plugins/main/bridge
```
Comment (LiliC) [inline README.md]: Did you mean `go get -u github.com/containernetworking/plugins/plugins/main/bridge`?

File: README.md [hunk 3]
```diff
@@ -74,51 +74,14 @@ cd cluster/images/hyperkube
...
+get get -u github.com/containernetworking/plugins/plugins/ipam/host-local
```
Comment (LiliC) [inline README.md]: Same as above.

**Pull request merged**